pyproject.toml

```diff
@@ -39,6 +41,7 @@ protoc-gen-python_betterproto = "betterproto.plugin:main"
 
 [tool.poetry.extras]
 compiler = ["black", "jinja2", "protobuf"]
+docs = ["toml", "sphinx"]
```

Comment:
what's the idea of adding sphinx as a extras dep here? does the rtd build process expect it?

Reply:
Should also add the following to  dev-dependencies to support building the docs in dev.

```toml
sphinx = "3.1.2"
sphinx-rtd-theme = "0.5.0"
```

Then adding the following to the `tool.poe.tasks` section will allow developers to build docs locally via `poe docs`

```toml
docs        = { cmd = "sphinx-build docs docs/build", help = "Build the sphinx docs"}
```

Reply:
It will just make the building a bit faster as it doesn't need compiler dependencies.